### PR TITLE
Add products edit screen column

### PR DESCRIPTION
**Pull request by unfulvio**
# Summary

Adds the `FB Sync Enabled` column to the products edit screen, displaying information with the sync handling for each corresponding product in the products list table.

## UI changes

none

### Setup 

You can use the QA cases in #15 to set a product meta flag to products: some products will have enabled sync, some disabled.

### Cases

- [x] Products with sync `enabled` should display "Enabled" in the corresponding row for the "FB Sync Enabled" column

- [x] Products with sync `disabled` should display "Disabled" in the corresponding row for the "FB Sync Enabled" column

### CODE REVIEW

File: includes/Admin.php [hunk 1]
```diff
@@ -23,6 +23,48 @@ class Admin {
...
+	 * Adds a column for Facebook Sync in the products edit screen.
+	 *
+	 * @internal
+	 *
+	 * @param array $columns array of keys and labels
+	 * @return array
+	 */
+	public function add_product_list_table_column( $columns ) {
+
+		$columns['facebook'] = __( 'FB Sync Status', 'facebook-for-woocommerce' );
```
Comment (wvega) [inline includes/Admin.php]: It seems the pitch was updated to use `FB Sync Enabled` as the name of the column and to keep the `FB Shop` column (now as `FB Shop Visibility`)

https://docs.google.com/document/d/1etRkNxkm2mfrOswHUCuVYTYe9cq0Artp0EIJ16yblF0/edit#heading=h.ivvxh9bh9mzw

File: facebook-commerce.php [hunk 2]
```diff
@@ -576,114 +567,37 @@ public function fb_new_product_tab_content() {
...
+	 * @return array
+	 */
+	public function fb_product_columns( $existing_columns ) {
 
-		// Verify that cart URL hasn't changed.  We do it here because this page
-		// is most likely to be visited (so it's a handy place to make the check)
-		$cart_url = get_option( self::FB_CART_URL );
-		if ( ! empty( $cart_url ) && ( wc_get_cart_url() !== $cart_url ) ) {
-			$this->display_warning_message(
-				'One or more of your products is using a
```
Comment (wvega) [inline facebook-commerce.php]: While I think this is not the place to do that verification, shouldn't we keep it around until we move it to a better place?

File: includes/Admin.php (hunk 1 above)
Comment (unfulvio) [reply includes/Admin.php]: ah sorry I must have missed that...

File: facebook-commerce.php (hunk 2 above)
Comment (unfulvio) [reply facebook-commerce.php]: wasn't there another story for that?

File: facebook-commerce.php [hunk 3]
```diff
@@ -317,17 +317,8 @@ public function __construct() {
 					1
 				);
 
-				  add_action( 'add_meta_boxes', array( $this, 'fb_product_metabox' ), 10, 1 );
+				add_action( 'add_meta_boxes', [ $this, 'fb_product_metabox' ] );
```
[the remark this answers is not in the document]
Comment (unfulvio) [reply facebook-commerce.php]: fixed in https://github.com/skyverge/facebook-for-woocommerce/pull/17/commits/c44e058bb3e93f058a332e60be7c11b3a364b0f7

File: includes/Admin.php (hunk 1 above)
Comment (unfulvio) [reply includes/Admin.php]: I've checked and this requires a new story: we should probably move the method that gets a Facebook product ID to the `Products` class first, there's a fair bit of logic there. For the scope of this PR I will just restore the code from the original version that added that column.

Comment (unfulvio) [reply includes/Admin.php]: @wvega I've updated the PR, and restored the former column, just changed its name for the time being -- thanks

File: facebook-commerce.php (hunk 2 above)
Comment (unfulvio) [reply facebook-commerce.php]: there is actually in implementation, but it's not created yet as a story in CH

Comment (wvega) [reply facebook-commerce.php]: You're right! I totally missed that one. Nothing to do here then.

**Comment by deivamagalhaes** [comment]
I've updated the column label and values as per the pitch changes in https://github.com/skyverge/facebook-for-woocommerce/pull/17/commits/48f1c4f618c5edf8e3a128cba09decc9219fd78c.

**Pull request merged**